examples/README.md

```diff
@@ -0,0 +1,23 @@
+#  Examples & Tutorials
+
+* Tutorial for simple bot [[notebook]](examples/gobot_tutorial.ipynb) [[colab]](https://colab.research.google.com/github/deepmipt/DeepPavlov/blob/master/examples/gobot_tutorial.ipynb)
```

Comment:
Aren't we already in the `examples` directory?
```suggestion
* Tutorial for simple bot [[notebook]](gobot_tutorial.ipynb) [[colab]](https://colab.research.google.com/github/deepmipt/DeepPavlov/blob/master/examples/gobot_tutorial.ipynb)
```

Reply:
True, i'll do the fix
It worked different when I tested links during readme editing.